src/app/pipes/math/bytes.spec.ts

```diff
@@ -14,14 +14,14 @@ describe('BytesPipe', () => {
   it('should return correct bytes format', () => {
     expect(pipe.transform(98)).toEqual('98 B');
     expect(pipe.transform(999)).toEqual('999 B');
-    expect(pipe.transform(1000)).toEqual('1 KB');
-    expect(pipe.transform(1e4)).toEqual('10 KB');
-    expect(pipe.transform(1e6 - 1)).toEqual('999.999 KB');
-    expect(pipe.transform(1e9 - 1)).toEqual('999.999999 MB');
-    expect(pipe.transform(1e9)).toEqual('1 GB');
-    expect(pipe.transform(1e12)).toEqual('1000 GB');
-    expect(pipe.transform(12345678)).toEqual('12.345678 MB');
-    expect(pipe.transform(12345678, 1)).toEqual('12.3 MB');
-    expect(pipe.transform(12345678, 2)).toEqual('12.35 MB');
+    expect(pipe.transform(1024)).toEqual('1 KB');
+    expect(pipe.transform(10240)).toEqual('10 KB');
+    expect(pipe.transform(1023488)).toEqual('999.5 KB');
+    expect(pipe.transform(15456864)).toEqual('14.740814208984375 MB');
+    expect(pipe.transform(4616548610)).toEqual('4.299496045917114 GB');
+    expect(pipe.transform(461654861500)).toEqual('429.94960505737276 GB');
+    expect(pipe.transform(12345678)).toEqual('11.77375602722168 MB');
```

Comment:
Let's set the default precision to 3, so it won't return values like in this test

Reply:
probably better to check the "full value" in the test than an round and we check with precision at the end